### DISCUSSION TITLE
Keeping pull request in sync with the base branch

### DISCUSSION THREAD

**Discussion by Bivek** [community/community#48774] [Pull Requests] [answered]
### Select Topic Area
Question

Pull Request Sync with base branch via Github UI no longer available

### Question
Keeping pull request in sync with the base branch in no loner available in Github PR UI

### Body
Hi!
Nowadays, I am not able to locate the github capability to sync the PR with the base branch which used to exists before.

Mainly this: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/keeping-your-pull-request-in-sync-with-the-base-branch

Is this feature currently removed due to some reason?

Thanks!

**Comment by Bivek** [comment, marked as answer]
The configuration **Always suggest updating pull request branches** configuration was not enabled that was the reason it was not available in GUI. This is now resolved. Thanks!

**Reply by queenofcorgis** [reply]
Thanks for coming back and sharing the solve @Bivek ⭐. Just want to mention that the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you to Pull Requests.
